pkg/cmd/docs-issue-generation/github.go

```diff
@@ -0,0 +1,294 @@
+// Copyright 2023 The Cockroach Authors.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+package main
+
+import (
+	"fmt"
+	"strings"
+	"time"
+)
+
+func searchCockroachPRs(startTime time.Time, endTime time.Time) ([]cockroachPR, error) {
+	prCommitsToExclude, err := searchJiraDocsIssues(startTime)
+	if err != nil {
+		fmt.Println(err)
+	}
+	hasNextPage, nextCursor, prs, err := searchCockroachPRsSingle(startTime, endTime, "", prCommitsToExclude)
+	if err != nil {
+		fmt.Println(err)
```

Comment:
As below: in `docs_issue_generation.go` line 68, the error is printed anyway. That means we don't have to repetitively print it here as well.

Same with the rest of the `fmt.Println(err)`: check whether they are actually needed. I would say that this is something you want to do *if and only if* we're just throwing away the error. If you're returning it, the job is done: someone else will handle it, print it, whatever.

Reply:
I've updated it so we only print the error message within the `main` function in `main.go`. It returns it everywhere else.